app/views/layouts/_beta_message.html.erb

```diff
@@ -1,8 +1,8 @@
 <div class="dgu-beta__banner">
   <div class="dgu-beta__message">
     <p>
-      We’ve been improving data.gov.uk to help you find and use open government data.
-      <%= link_to "Discover what’s changed", about_path %> and <%= link_to "get in touch", support_path %> to give us your feedback.
+      We've been improving data.gov.uk to help you find and use open government data.
```

Comment:
I think that's not what you want. We are supposed to use smart quotes in text.

Reply:
Ah ok, sorry, my mistake.